todo_app/items_test.py

```diff
@@ -0,0 +1,121 @@
+
+from todo_app.data.ViewModel import ViewModel
+from datetime import date
+from flask import Flask,request, render_template,redirect,url_for
+import pytest
+import requests
+import os
+from todo_app.data.ToDo import ToDo
+
+toDoId = os.environ['TOID']
+doingId = os.environ['DOINGID']
+doneId = os.environ['DONE']
+
+
+def getItems(idList):
```

Comment:
If this is your unit test, we really want it to focus closely on a single unit, in this case the ViewModel class, whereas here you're testing a function that only exists for the test file.

An example of a unit test for the show_all_done_items might look like below

```python
def test_show_all_done_items_true_if_1_done_item():
	to_do_items = []
	doing_items = []
	done_items = [
		ToDo('id', 'name', date.today()),
	]
	view_model = ViewModel(to_do_items, doing_items, done_items)

	assert view_model.show_all_done_items() == True
```

This tight focus on a single unit is really important - we avoid the test from worrying about environment variables, communicating with trello etc.

Reply:
I have added the test, thank you

Reply:
These look much more like unit tests, so we can now get rid of this getItems function